pkg/sass-parser/lib/src/statement/index.ts

```diff
@@ -30,14 +37,29 @@ export type AnyStatement = Root | Rule | GenericAtRule;
  *
  * @category Statement
  */
-export type StatementType = 'root' | 'rule' | 'atrule';
+export type StatementType =
+  | 'root'
+  | 'rule'
+  | 'atrule'
+  | 'comment'
+  | 'debug-rule'
+  | 'each-rule'
+  | 'for-rule'
+  | 'error-rule';
 
 /**
  * All Sass statements that are also at-rules.
  *
  * @category Statement
  */
-export type AtRule = GenericAtRule;
+export type AtRule = DebugRule | EachRule | ErrorRule | ForRule | GenericAtRule;
+
+/**
+ * All Sass statements that are comments.
```

Comment:
Is this plural because we will add a separate type for silent comments?

Reply:
Exactly.